test/test_package.py

```diff
@@ -158,11 +158,6 @@ def test_validate_package(self):
         pack.validate(warnings=warnings)
         self.assertIn('naming conventions', warnings[0])
 
-        pack.name = 'bar-bza'
-        warnings = []
-        pack.validate(warnings=warnings)
-        self.assertIn('naming conventions', warnings[0])
-
```

Comment:
Instead of removing this test can you please update it to ensure that it is a valid package name now.